VirtusLab-Open-Source/strapi-plugin-navigation
# GraphQL: remove unnecessary `attributes` level in related item schema

## Ticket

https://github.com/VirtusLab/strapi-plugin-navigation/issues/485

## Summary

This PR removes the `attributes` property in the GraphQL schema. This property is no longer used in Strapi v5.

## Test Plan

1. Create a menu with INTERNAL links.
2. Query this menu with GraphQL and try to read the `related` properties.
3. Observe that before this PR, it was impossible to read the properties of the related item and after this PR it is possible.
